python/graphstorm/model/edge_decoder.py

```diff
@@ -1116,29 +1116,33 @@ def get_relembs(self):
         return self._w_relation.weight, self.etype2rid
 
 class LinkPredictRotatEDecoder(LinkPredictMultiRelationLearnableDecoder):
-    r""" Decoder for link prediction using the RotatE as the score function.
+    r"""
+    .. versionadded:: 0.4
```

Comment:
Let's separate the docstring update PR with the New evaluator PR.

Reply:
will submit another PR